run/types/DeviceWrapper.ts

```diff
@@ -933,12 +1080,288 @@ export class DeviceWrapper implements SharedDeviceInterface {
     await this.setValueImmediate(text, element.ELEMENT);
   }
 
+  public async getAttribute(
+    attribute: string,
+    elementId: string
+  ): Promise<string> {
+    return this.toShared().getAttribute(attribute, elementId);
+  }
+
+  public async disappearRadioButtonSelected(timeOption: DMTimeOption) {
+    try {
+      const radioButton = await this.findElementByXPath(
+        `//*[./*[@name='${timeOption}']]/*[2]`
+      );
+
+      const attr = await this.getAttribute("value", radioButton.ELEMENT);
+      if (attr === "selected") {
+        console.log("Great success - default time is correct");
+      } else {
+        throw new Error("Dammit - default time was not correct");
+      }
+    } catch (e) {
+      console.log(`Couldn't find radioButton ${timeOption}`);
+    }
+  }
+  // TODO FIX UP THIS FUNCTION
+  public async sendImage(
+    platform: SupportedPlatformsType,
+    message?: string,
+    community?: boolean
+  ) {
+    if (platform === "ios") {
```

Comment:
```suggestion
if(this.isIos())
```

same for other calls